goManual/asciidoc/session-api.adoc

```diff
@@ -6,9 +6,14 @@ include::{common-content}/session-api.adoc[tag=abstract]
 Sessions provide a "classic" *_blocking style_* API for Cypher execution.
 In general, sessions provide the easiest programming style to work with since API calls are executed in a strictly sequential fashion.
 
+[[go-driver-simple-sessions]]
+== Simple sessions
+
+include::{common-content}/session-api.adoc[tag=simple-sessions]
+
```

Comment:
This section header was removed previously (#175) as Go only has simple sessions, and therefore we don't need to refer to them as "simple". This had the knock on effect of needing to drop the depth of header on the subsequent sections

Reply:
Good point. My focus was on removing the build error, but I hadn't allowed for that consideration. I think the change you made previously was better, so I've restored that and updated the nav to remove errors.